minerl/herobraine/hero/handlers/agent/start.py

```diff
@@ -93,3 +93,61 @@ def __init__(self, x, y, z, yaw):
         self.y = y
         self.z = x
         self.yaw = yaw
+
+
+class StartingHealthAgentStart(Handler):
+    def to_string(self) -> str:
+        return "starting_health_agent_start"
+
+    def xml_template(self) -> str:
+        if self.health is None:
+            return str(
+                """<StartingHealth maxHealth="{{ max_health }}"/>"""
+            )
+        else:
+            return str(
+                """<StartingHealth maxHealth="{{ max_health }}" health="{{ health }}"/>"""
+            )
+
+    def __init__(self, max_health: float = 20, health: float = None):
+        """Sets the starting health of the agent.
+
+        For example:
+
+            starting_health = StartingHealthAgentStart(2.5)
+
+        Args:
+            max_health: The maximum amount of health the agent can have
+            health: The amount of health the agent starts with (max_health if not specified)
+        """
+        self.health = health
+        self.max_health = max_health
```

Comment:
I guess these defaults effectively make both values optional.

Reply:
To be clear this is also true in the xml.

Reply:
Yep, you can ignore that comment.